test/kale/dry_run_test.clj

```diff
@@ -94,21 +93,3 @@
                     (do (sut/dry-run {}
                                      ["dry-run" "test-file.html"] [])
                         (slurp "converted/test-file.html.json")))))))))
-
-(deftest dry-run-failure
```

Comment:
Because it was hard? (and then I forgot about it) I'll see if I can get it back in there.